### PR TITLE
[chore] Added Jeremy Hicks (greatestusername) to codeowner list

### PR DESCRIPTION
**Pull request by shalper2**
**Description:** Added user Jeremy Hicks (@greatestusername) to codeowner list

**Link to tracking Issue:** none

**Testing:** none

**Documentation:** Updated documentation and ran mdatagen to update relevant files. Some newlines were added by mdatagen.

### CODE REVIEW

File: .chloggen/splunkent-add-codeowner.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```
Comment (codeboten) [inline .chloggen/splunkent-add-codeowner.yaml:1]: no need for a changelog for this, please remove and i'll add a label to skip changelog checks 👍🏻

Comment (shalper2) [reply .chloggen/splunkent-add-codeowner.yaml]: heard! wasn't sure figured better safe than sorry.

**Comment by crobert-1** [comment]
@greatestusername Can you approve here too?

**Review by crobert-1 — APPROVED** [review]

**Comment by atoulme** [comment]
you also need to add the codeowner to `.github/CODEOWNERS`.

**Comment by atoulme** [comment]
also we need to add @greatestusername to the OpenTelemetry membership. @greatestusername please go ahead and open this request: https://github.com/open-telemetry/community/issues/new?assignees=&labels=area%2Fgithub-membership&projects=&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E

**Comment by crobert-1** [comment]
> you also need to add the codeowner to `.github/CODEOWNERS`.

This can be done with `make gengithub`, right?

**Comment by atoulme** [comment]
> > you also need to add the codeowner to `.github/CODEOWNERS`.
> 
> This can be done with `make gengithub`, right?

you also need to set a github token. I might bring back the members cache rather than having github API called every time.

**Comment by greatestusername-splunk** [comment]
> also we need to add @greatestusername to the OpenTelemetry membership. @greatestusername please go ahead and open this request: https://github.com/open-telemetry/community/issues/new?assignees=&labels=area%2Fgithub-membership&projects=&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E

https://github.com/open-telemetry/community/issues/1819
Let me know if there is anything else you need me to do.

Thanks!!

**Comment by greatestusername** [comment]
> also we need to add @greatestusername to the OpenTelemetry membership. @greatestusername please go ahead and open this request: https://github.com/open-telemetry/community/issues/new?assignees=&labels=area%2Fgithub-membership&projects=&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E

https://github.com/open-telemetry/community/issues/1819
Let me know if there is anything else you need me to do.

Thanks!!

**Comment by greatestusername** [comment]
I am now a member and `make gengithub` should work now I think?
Thanks all!!!

**Comment by shalper2** [comment]
@greatestusername has been added to the codeowners list hooray

**Comment by atoulme** [comment]
just run `make generate` at the root to get the README regenerated too...

**Review by atoulme — APPROVED** [review]

**Pull request merged**